docs/src/content/docs/architecture/work-sources.md

```
@@ -0,0 +1,585 @@
+### WorkItem
+
+Every work source adapter normalizes external items into a common `WorkItem` structure. This allows the scheduler and runner to handle work items uniformly regardless of their origin.
+
+```typescript
+interface WorkItem {
+  id: string;             // Source-prefixed ID (e.g., "github-42")
+  source: string;         // Adapter type ("github")
+  externalId: string;     // ID in the external system ("42")
+  title: string;          // Human-readable title
+  description: string;    // Full body/description
+  priority: WorkItemPriority;  // "critical" | "high" | "medium" | "low"
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)